src/client/components/OrganizationTile.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import { Button as AragonButton, Text, theme } from '@aragon/ui'
+import { IconTrash } from './styled-components'
+
+const OrganizationTile = ({ organizationData, removeItem }) => (
+  <OrganizationItem>
+    <div>
+      <Text.Block size="large">{organizationData.address}</Text.Block>
+    </div>
```

Comment:
We might be able to get rid of the wrapping `div` here

Reply:
Removed, but it also will be redesigned when we actually can add Organizations.